fili-wikipedia-example/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM druidio/example-cluster:latest
+LABEL maintainer="https://groups.google.com/forum/#!forum/fili-developers"
+
+# fili github information
+ENV GITHUB_OWNER=yahoo BRANCH=master
+
+RUN apt-get update && apt-get install -y --no-install-recommends git
+
+RUN wget -q -O - http://archive.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz | tar -xzf - -C /usr/local \
+      && ln -s /usr/local/apache-maven-3.2.5 /usr/local/apache-maven \
+      && ln -s -f /usr/local/apache-maven/bin/mvn /usr/local/bin/mvn
+
+# fili code goes here
+RUN mkdir -p /usr/local/fili
+
+# update in case of new commits
+ADD https://api.github.com/repos/$GITHUB_OWNER/fili/git/refs/heads/$BRANCH \
+fili-branch.json
+RUN git clone -q --branch ${BRANCH} --depth 1 \
+https://github.com/$GITHUB_OWNER/fili.git /usr/local/fili
+
+WORKDIR /usr/local/fili
+
+# install Fili in the container
+RUN mvn -U -B install -DskipTests=true -Dmaven.javadoc.skip=true -Dcheckstyle.skip
+
+EXPOSE 9998
+
+ENTRYPOINT export HOSTIP="$(resolveip -s $HOSTNAME)" \
```

Comment:
Sounds good. I'm working on the shell script right now. Quick question, should I use port 3001 in the while loop instead of 8081? I tried going to 
```http://localhost:8081/druid/coordinator/v1/datasources``` 
while the container was running but I couldn't connect to port 8081. It seemed to work when I did this instead:
```http://localhost:3001/druid/coordinator/v1/datasources```